app/app.js

```diff
@@ -111,6 +111,10 @@ monitor.run();
 
 // helpers
 function tryExpandEnvironmentVariables(monitorConfiguration, serviceConfiguration) {
+    if (process.env.TOKEN) {
```

Comment:
Please revert this change. This should happen automatically by `tryExpandEnvironmentVariable`, if `expandEnvironmentVariables` is set to `true`.